From 2149b2aa70e12a7b88c6296e10c05c45b23e04cc Mon Sep 17 00:00:00 2001
From: Michal Simek <monstr@monstr.eu>
Date: Tue, 3 Apr 2012 12:53:17 +0200
Subject: [PATCH] arm: gic: Fix gic_set_cpu

gic_set_cpu can't be placed in init sections because
there are removed and this function can't be used by modules.

Should be the part of this patch:
"arm: zynq: gic: Use OF initialization"
(sha1: bfc0225ab4ce50abcd29b0dde5430c88f95d6f39)

Also coding style cleanup.

Signed-off-by: Michal Simek <monstr@monstr.eu>
---
 arch/arm/common/gic.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/common/gic.c b/arch/arm/common/gic.c
index 60692ae..2998d9e 100644
--- a/arch/arm/common/gic.c
+++ b/arch/arm/common/gic.c
@@ -258,14 +258,14 @@ static int gic_set_affinity(struct irq_data *d, const struct cpumask *mask_val,
 	return IRQ_SET_MASK_OK;
 }
 
-void __init gic_set_cpu(unsigned int cpu, unsigned int irq)
+void gic_set_cpu(unsigned int cpu, unsigned int irq)
 {
 	struct irq_data *d = irq_get_irq_data(irq);
 	struct cpumask mask;
 
 	cpumask_clear(&mask);
 	cpumask_set_cpu(cpu, &mask);
-        gic_set_affinity (d, &mask, true);
+	gic_set_affinity(d, &mask, true);
 }
 EXPORT_SYMBOL(gic_set_cpu);
 #endif
-- 
1.7.0.4

